dali/operators/displacement/rotate_params.h

```diff
@@ -141,8 +188,42 @@ class RotateParamProvider<Backend, 2, BorderType>
     }
   }
 
+  template <int N, typename T>
+  void Collect(std::vector<vec<N, T>> &v, const std::string &name, bool required) {
```

Comment:
Can we move to some utils? It may be reused by other ops as well?

Reply:
In another PR maybe...